package.json

```diff
@@ -13,7 +13,6 @@
   "scripts": {
     "cypress:open": "cypress open",
     "cypress:run:chrome": "cypress run --headless --browser chrome",
-    "cypress:run:edge": "cypress run --headless --browser edge",
```

Comment:
Why was this removed?

Reply:
Oh let me fix that. I think I hit undo and went back to an old version of the file.